File: DK_PR_Front/ClientApp/src/app/models/post.ts

    export class Post
    {
      private postId: number = -1;
      public userName: string = "";
      public emojiName: string = "";
      public date: string = "";
      public postText: string = "";

      public setPost(postText: string, userName: string, emojiName: string, date: string) {
        this.postId++;
        this.userName = userName;
        this.emojiName = emojiName;
        this.date = date;
        this.postText = postText;

      }

      public getPostId(): number {
        return this.postId;
      }

    }

File: DK_PR_Front/Models/Post.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DK_PR_Front.Models
{
    [BsonIgnoreExtraElements]

    public class Post
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]

        public ObjectId Id { get; set; }

        [BsonElement("PostId")]
        public int PostId { get; set; }

        [BsonElement("UserName")]
        public string UserName { get; set; }

        [BsonElement("Date")]
        public DateTime Date { get; set; }

        [BsonElement("EmojiName")]
        public string EmojiName { get; set; }

        [BsonElement("PostText")]
        public string PostText { get; set; }


    }
}
